courses/machine_learning/deepdive2/text_classification/labs/classify_text_with_bert.ipynb

```diff
@@ -0,0 +1,1190 @@
+{
```

Comment:
<p>Nit: add Learning Objectives heading for consistency with ASL labs above "in this notebook" line.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/1412/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>